src/TTGO_T-Beam_LoRa_APRS.ino

```diff
@@ -832,19 +843,36 @@ void loop() {
       delay(300);
       time_delay = millis() + 1500;
       if(digitalRead(BUTTON)==HIGH){
-        if(gps_state == true && gps.location.isValid()){
-            writedisplaytext("((MAN TX))","","","","","");
-            sendpacket();
-        }else{
-            writedisplaytext("((FIX TX))","","","","","");
-            sendpacket();
+        if (!tempOled) {
+        // turn ON OLED temporary
+        enableOled();
+        //----------------------------
+        } else {
+          if(gps_state == true && gps.location.isValid()){
+              writedisplaytext("((MAN TX))","","","","","");
+              sendpacket();
+          }else{
+              writedisplaytext("((FIX TX))","","","","","");
+              sendpacket();
+          }
         }
         key_up = true;
       }
     }
   }
 
+  // Only wake up OLED when necessary
+  display.dim(!tempOled);
+
+  if (tempOled && millis()>= oled_timer) {
+    tempOled = false;
+  }
+  //------------------------------------
+
```

Comment:
This function wakes up the display when any button is pressed and turns it off after a fixed delay